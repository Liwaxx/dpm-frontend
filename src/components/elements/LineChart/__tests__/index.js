import React from 'react';
import renderer from 'react-test-renderer';
import Linechart from '../index';

jest.mock('../../Text');
jest.mock('../styles');

describe('PieChart', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Linechart />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with legend', () => {
    const legend = ['tes'];
    const tree = renderer
      .create(<Linechart legend={legend}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});