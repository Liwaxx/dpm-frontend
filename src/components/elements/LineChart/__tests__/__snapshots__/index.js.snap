// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PieChart renders correctly 1`] = `
<div
  className="MuiGrid-container-9"
>
  <div
    className="MuiGrid-item-10 MuiGrid-grid-xs-2-39"
  >
    <div
      className="Component-span-3"
    >
      
    </div>
  </div>
  <div
    className="MuiGrid-item-10 MuiGrid-grid-xs-10-47"
  />
</div>
`;

exports[`PieChart renders with legend 1`] = `
<div
  className="MuiGrid-container-9"
>
  <div
    className="MuiGrid-item-10 MuiGrid-grid-xs-2-39"
  >
    <div
      className="Component-span-3"
    />
  </div>
  <div
    className="MuiGrid-item-10 MuiGrid-grid-xs-10-47"
  />
</div>
`;
